tests/e2e/contracts/intelligent_oracle.py

```
@@ -0,0 +1,369 @@
+        )
+
+        title = self.title
+        description = self.description
+        potential_outcomes = list(self.potential_outcomes)
+        rules = list(self.rules)
+        earliest_resolution_date = self.earliest_resolution_date
+
+        for resource_url in resources_to_check:
+
+            def evaluate_single_source() -> str:
+                resource_web_data = gl.get_webpage(resource_url, mode="text")
```

Ruff (0.11.9)

`gl` may be undefined, or defined from star imports

(F405)